fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -0,0 +1,3973 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
if this file is most from other files, you should keep the original license header.

Reply:
1